include/circt/Conversion/Passes.td

```diff
@@ -337,6 +337,19 @@ def HandshakeRemoveBlock : Pass<"handshake-remove-block-structure", "handshake::
   let constructor = "circt::createHandshakeRemoveBlockPass()";
 }
 
+def InsertMergeBlocks : Pass<"insert-merge-blocks", "::mlir::ModuleOp"> {
+  let summary = "Insert explicit merge blocks";
+  let description = [{
+    Inserts additional merge blocks for each block with more than two successors.
+    The order the successors are merged together mirrors the order the different
+    control paths were created.
```

Comment:
Additionally, a comment on _why_ this might be needed (this, essentially, defines a canonical form for a CFG that facilitates further transformation).